packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanTreeOffset.css

```diff
@@ -39,15 +39,11 @@ limitations under the License.
 }
 
 .SpanTreeOffset--indentGuide.is-active {
-  /* The size of the indentGuide is based off of the iconWrapper */
-  padding-right: calc(0.5rem + 11px);
-  border-left: 0px;
+  border-left: 3px solid darkgrey;
 }
 
 .SpanTreeOffset--indentGuide.is-active:before {
-  content: '';
-  padding-left: 3px;
-  background-color: darkgrey;
+  background-color: transparent;
```

Comment:
I believe a psuedoelement without `content` is not generated at all.

Please remove this `:before` entirely if it is not necessary.

Reply:
padding is a part of element size https://github.com/jaegertracing/jaeger-ui/blob/09d763f6b1d262118c15244c49c16d326e6b518d/packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanTreeOffset.css#L37 this is needed to show tiny lines (lines when `:not(:hover)`)